packages/IMAPClient-Tests.package/ICEmailTest.class/instance/testRetrieveBody.st

```diff
@@ -4,6 +4,8 @@ testRetrieveBody
 	| folder endpoint |
 	
 	endpoint := ICMockEndpoint new.
+	Transcript show: 'Jetzt geht es los'; cr.
+	Transcript show: self email body; cr.
```

Comment:
These lines might be from the debugging phase, but before merging, please remove Transcript output.